.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+<!-- Before creating an issue please make sure you are using the latest version of frontend-maven-plugin. -->
```

Comment:
People should also search in the issue/PR archive to check whether anybody else has already requested the same.